node/src/components/consensus/highway_core/finality_detector/rewards.rs

```diff
@@ -0,0 +1,337 @@
+use tracing::trace;
```

Comment:
Thanks, good catch. File and the associated code is removed in [this](https://github.com/casper-network/casper-node/pull/4228/commits/48c9d1e1fa45af727b19df2d6d7c8c9ddf0836c1) commit, **however** I added a `TODO` to actually take care of the private chain support in the 2.0 rewards scheme. Please have a look @AlexanderLimonov as this may require additional ticket related to 2.0 rewards.

Reply:
One could use the new rewards just as one would old rewards for private chains